gulp test:local fails all some tests if base directory is not named 'polymer-starter-kit'

I have another project based on the PSK 1.0, all tests pass without an issue no matter what the directory name is, I might try to update it in a separate directory to see how it behaves.

Setup:

Ubuntu 14.04.3 LTS
node v0.12.7
npm 2.14.1
bower 2.14.1
zsh 5.0.2 (x86_64-pc-linux-gnu) (plus Prezto)
Google Chrome Version 45.0.2454.93 (64-bit)
Firefox 40.0.3

But in this case I did all from zero:
```bash
$ git clone https://github.com/PolymerElements/polymer-starter-kit.git
$ cd polymer-starter-kit
$ npm install -g gulp bower && npm install && bower install
```

Then did the following:
**gulp && gulp test:local**, changed directory name to **psk110**, tried **gulp && gulp test:local** again, this time all tests fail, then I changed the directory back to **polymer-starter-kit** again, all tests passed:
```bash
~/projects/psk/polymer-starter-kit master 1m 28s
❯ gulp && gulp test:local
[09:53:53] Using gulpfile ~/projects/psk/polymer-starter-kit/gulpfile.js
[09:53:53] Starting 'clean'...
[09:53:53] Finished 'clean' after 302 ms
[09:53:53] Starting 'default'...
[09:53:53] Starting 'copy'...
[09:53:53] Starting 'styles'...
[09:53:54] styles all files 98 B
[09:53:54] Finished 'styles' after 582 ms
[09:53:55] copy all files 10.98 MB
[09:53:55] Finished 'copy' after 1.38 s
[09:53:55] Starting 'elements'...
[09:53:55] Finished 'elements' after 4.46 ms
[09:53:55] Starting 'jshint'...
[09:53:55] Starting 'images'...
[09:53:55] Starting 'fonts'...
[09:53:55] Starting 'html'...
[09:53:56] Finished 'fonts' after 520 ms
[09:53:58] Finished 'jshint' after 2.8 s
[09:53:58] images all files 35.41 kB
[09:53:58] Finished 'images' after 2.61 s
[09:53:58] html all files 51.25 kB
[09:53:58] Finished 'html' after 2.22 s
[09:53:58] Starting 'vulcanize'...
[09:54:06] Finished 'vulcanize' after 8.25 s
[09:54:06] Starting 'rename-index'...
[09:54:06] Finished 'rename-index' after 4.79 ms
[09:54:06] Finished 'default' after 12 s
[09:54:08] Using gulpfile ~/projects/psk/polymer-starter-kit/gulpfile.js
[09:54:08] Starting 'wct:local'...
Installing and starting Selenium server for local browsers
Selenium server running on port 48822
Web server running on port 2000 and serving from /home/ricardo/projects/psk/polymer-starter-kit
chrome 45                Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=0
chrome 45                Tests passed
firefox 40               Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=1
firefox 40               Tests passed
Test run ended with great success

chrome 45 (2/0/0)                       firefox 40 (2/0/0)                    
[09:54:25] Finished 'wct:local' after 16 s
[09:54:25] Starting 'test:local'...
[09:54:25] Finished 'test:local' after 10 μs
                                                                                                                                                                                                                
~/projects/psk/polymer-starter-kit master 35s
❯ cd ..
                                                                                                                                                                                                                
~/projects/psk
❯ rm -rf psk110
                                                                                                                                                                                                                
~/projects/psk 13s
❯ mv polymer-starter-kit psk110

~/projects/psk
❯ cd psk110
                                                                                                                                                                                                                
~/projects/psk/psk110 master
❯ gulp && gulp test:local
[09:55:27] Using gulpfile ~/projects/psk/psk110/gulpfile.js
[09:55:27] Starting 'clean'...
[09:55:28] Finished 'clean' after 318 ms
[09:55:28] Starting 'default'...
[09:55:28] Starting 'copy'...
[09:55:28] Starting 'styles'...
[09:55:28] styles all files 98 B
[09:55:28] Finished 'styles' after 569 ms
[09:55:29] copy all files 10.98 MB
[09:55:29] Finished 'copy' after 1.38 s
[09:55:29] Starting 'elements'...
[09:55:29] Finished 'elements' after 6.75 ms
[09:55:29] Starting 'jshint'...
[09:55:29] Starting 'images'...
[09:55:29] Starting 'fonts'...
[09:55:29] Starting 'html'...
[09:55:30] Finished 'fonts' after 409 ms
[09:55:31] Finished 'jshint' after 2.57 s
[09:55:31] images all files 35.41 kB
[09:55:31] Finished 'images' after 2.36 s
[09:55:31] html all files 51.25 kB
[09:55:31] Finished 'html' after 2.01 s
[09:55:31] Starting 'vulcanize'...
[09:55:40] Finished 'vulcanize' after 8.39 s
[09:55:40] Starting 'rename-index'...
[09:55:40] Finished 'rename-index' after 2.95 ms
[09:55:40] Finished 'default' after 12 s
[09:55:42] Using gulpfile ~/projects/psk/psk110/gulpfile.js
[09:55:42] Starting 'wct:local'...
Installing and starting Selenium server for local browsers
Selenium server running on port 45563
Web server running on port 2000 and serving from /home/ricardo/projects/psk/psk110
chrome 45                Beginning tests via http://localhost:2000/components/psk110/generated-index.html?cli_browser_id=0
404 GET /components/psk110/app/bower_components/polymer/polymer.html
chrome 45                ✖ Test Suite Initialization

  Polymer is not defined
    <unknown> at   <unknown> at /components/psk110/app/elements/my-greeting/my-greeting.html:30:0
    <unknown> at   <unknown> at /components/psk110/app/elements/my-greeting/my-greeting.html:41:0

chrome 45                ✖ app/test/my-greeting-basic.html » my-greeting tests » Welcome!

  Cannot read property 'textContent' of null
    <unknown> at   Context.<anonymous> at my-greeting-basic.html:45:0

404 GET /components/psk110/app/bower_components/polymer/polymer.html
chrome 45                ✖ Test Suite Initialization

  Polymer is not defined
    <unknown> at   <unknown> at /components/psk110/app/elements/my-list/my-list.html:30:0
    <unknown> at   <unknown> at /components/psk110/app/elements/my-list/my-list.html:49:0

chrome 45                ✖ app/test/my-list-basic.html » my-list tests » Item lengths should be equalled

  expected 6 to equal 0
    <unknown> at   <unknown> at my-list-basic.html:59:0

chrome 45                Tests failed: 4 failed tests
firefox 40               Beginning tests via http://localhost:2000/components/psk110/generated-index.html?cli_browser_id=1
404 GET /components/psk110/app/bower_components/polymer/polymer.html
firefox 40               ✖ app/test/my-greeting-basic.html » my-greeting tests » Welcome!

  header is null
    <unknown> at   <unknown> at my-greeting-basic.html:45:0

404 GET /components/psk110/app/bower_components/polymer/polymer.html
firefox 40               ✖ app/test/my-list-basic.html » my-list tests » Item lengths should be equalled

  AssertionError: expected 6 to equal 0 (:0)
    <unknown> at   <unknown> at /components/mocha/mocha.js:12308:0

firefox 40               Tests failed: 2 failed tests
Test run ended in failure: 4 failed tests, 2 failed tests

chrome 45 (0/0/4)                       firefox 40 (0/0/2)                    
[09:55:59] 'wct:local' errored after 17 s
[09:55:59] Error: 4 failed tests, 2 failed tests
                                                                                                                                                                                                                
~/projects/psk/psk110 master 35s
❯ cd ..
                                                                                                                                                                                                                
~/projects/psk
❯ mv psk110 polymer-starter-kit
                                                                                                                                                                                                                
~/projects/psk
❯ cd polymer-starter-kit
                                                                                                                                                                                                                
~/projects/psk/polymer-starter-kit master
❯ gulp && gulp test:local
[10:04:54] Using gulpfile ~/projects/psk/polymer-starter-kit/gulpfile.js
[10:04:54] Starting 'clean'...
[10:04:54] Finished 'clean' after 248 ms
[10:04:54] Starting 'default'...
[10:04:54] Starting 'copy'...
[10:04:54] Starting 'styles'...
[10:04:55] styles all files 98 B
[10:04:55] Finished 'styles' after 588 ms
[10:04:56] copy all files 10.98 MB
[10:04:56] Finished 'copy' after 1.37 s
[10:04:56] Starting 'elements'...
[10:04:56] Finished 'elements' after 4.82 ms
[10:04:56] Starting 'jshint'...
[10:04:56] Starting 'images'...
[10:04:56] Starting 'fonts'...
[10:04:56] Starting 'html'...
[10:04:57] Finished 'fonts' after 410 ms
[10:04:58] Finished 'jshint' after 2.6 s
[10:04:58] images all files 35.41 kB
[10:04:58] Finished 'images' after 2.38 s
[10:04:58] html all files 51.25 kB
[10:04:58] Finished 'html' after 2 s
[10:04:58] Starting 'vulcanize'...
[10:05:06] Finished 'vulcanize' after 8.27 s
[10:05:06] Starting 'rename-index'...
[10:05:06] Finished 'rename-index' after 3.61 ms
[10:05:06] Finished 'default' after 12 s
[10:05:09] Using gulpfile ~/projects/psk/polymer-starter-kit/gulpfile.js
[10:05:09] Starting 'wct:local'...
Installing and starting Selenium server for local browsers
Selenium server running on port 59171
Web server running on port 2000 and serving from /home/ricardo/projects/psk/polymer-starter-kit
chrome 45                Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=0
chrome 45                Tests passed
firefox 40               Beginning tests via http://localhost:2000/components/polymer-starter-kit/generated-index.html?cli_browser_id=1
firefox 40               Tests passed
Test run ended with great success

chrome 45 (2/0/0)                       firefox 40 (2/0/0)                    
[10:05:24] Finished 'wct:local' after 15 s
[10:05:24] Starting 'test:local'...
[10:05:24] Finished 'test:local' after 9.77 μs
                                                                                                                                                                                                                
~/projects/psk/polymer-starter-kit master 32s
❯ 
```

--- a/wct.conf.js
+++ b/wct.conf.js
@@ -1,0 +1,18 @@
+var path = require('path');
+
+var ret = {
+  'suites': ['app/test'],
+  'webserver': {
+    'pathMappings': []
+  }
+};
+
+var mapping = {};
+var rootPath = (__dirname).split(path.sep).slice(-1)[0];
+
+mapping['/components/' + rootPath  +
+'/app/bower_components'] = 'bower_components';
+
+ret.webserver.pathMappings.push(mapping);
+
+module.exports = ret;

--- a/wct.conf.json
+++ b/wct.conf.json
@@ -1,8 +1,0 @@
-{
-  "suites": ["app/test"],
-  "webserver": {
-    "pathMappings": [
-      {"/components/polymer-starter-kit/app/bower_components": "bower_components"}
-    ]
-  }
-}

